easybuild/easyblocks/r/rosetta.py

```diff
@@ -121,9 +121,12 @@ def configure_step(self):
         self.log.debug("List of extra environment variables to pass down: %s" % str(env_vars))
 
         # create user.settings file
-        paths = os.getenv('PATH').split(':')
-        ld_library_paths = os.getenv('LD_LIBRARY_PATH').split(':')
-        cpaths = os.getenv('CPATH').split(':')
+        paths = os.getenv('PATH')
+        paths = paths.split(':') if paths else []
+        ld_library_paths = os.getenv('LD_LIBRARY_PATH')
+        ld_library_paths = ld_library_paths.split(':') if ld_library_paths else []
+        cpaths = os.getenv('CPATH')
+        cpaths = cpaths.split(':') if cpaths else []
```

Comment:
```suggestion
        paths = os.getenv('PATH', '').split(':')
        ld_library_paths = os.getenv('LD_LIBRARY_PATH', '').split(':')
        cpaths = os.getenv('CPATH', '').split(':')
```

Reply:
Though that gives `['']`, so ignore me.

Reply:
@branfosj That's not exactly the same:

```
>>> os.getenv('NOTSET', '').split(':')
['']
```

Using a list with a single empty string element vs an empty list is very different...

Reply:
Yes, hence me dismissing my requested change.

I'd forgotten that Python treats `split()` with and without a delimiter differently:
```
>>> os.getenv('BAR', '').split(':')
['']
>>> os.getenv('BAR', '').split()
[]
```